fee_allocator/accounting/distribution.py

```diff
@@ -171,6 +174,28 @@ def re_distribute_incentives(
             incentives[pool_id]['bal_incentives'] -= aura_incentives_to_increase
     return incentives
 
+def add_last_join_exit(incentives: Dict[str, Dict], chain: Chains, alertTimeStamp: Optional[int] = None) -> Dict[str, Dict]:
+    """
+    adds last_join_exit for each pool in the incentives list for reporting.
+    Returns the same thing as inputed with the additional field added for each line
+    """
+    q = BalPoolsGauges(chain.value)
+    results = {}
+    for pool_id, incentive_data in incentives.items():
+        results[pool_id] = incentive_data
+        timestamp = q.get_last_join_exit(pool_id)
+        try:
+            timestamp = q.get_last_join_exit(pool_id)
+        except:
+            results[pool_id]["last_join_exit"] = "Error fetching"
+            continue
+        gmt_time = datetime.datetime.utcfromtimestamp(timestamp)
+        human_time = gmt_time.strftime('%Y-%m-%d %H:%M:%S GMT')
```

Comment:
GMT might be a bit annoying for time sorting. I suggest to use something else like YYYY-MM-DDTHH:MM:SSZ so the output is similar to 2023-03-23T18:10:55+00:00 as an example

Reply:
Sounds good.  Do you want the specific/exact time string you have specified to make life easier, or can I just change  GMT here to %Z?

Reply:
The latter sounds good